### PR TITLE
test: 💍 with v5.0.3 chain snapshot

### PR DESCRIPTION
**Pull request by polymath-eric**
### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

File: tests/entities/__snapshots__/assets.test.ts.snap [hunk 1]
```diff
@@ -105,20 +105,13 @@ Object {
...
           "__typename": "TransferManagersConnection",
-          "nodes": Array [
-            Object {
-              "__typename": "TransferManager",
-              "exemptedEntities": Array [],
-              "type": "Count",
-              "value": 10,
-            },
-          ],
+          "nodes": Array [],
```
Comment (polymath-eric) [inline tests/entities/__snapshots__/assets.test.ts.snap:114]: *empty

File: tests/entities/__snapshots__/claim.test.ts.snap [hunk 2]
```diff
@@ -4,24 +4,7 @@ exports[`claimScopes should return a list of scopes for given identity 1`] = `
...
-      Object {
-        "__typename": "ClaimScope",
-        "scope": Object {
-          "type": "Ticker",
-          "value": "5TICKER",
-        },
-        "ticker": "5TICKER",
-      },
-    ],
+    "nodes": Array [],
```
Comment (polymath-eric) [inline tests/entities/__snapshots__/claim.test.ts.snap:7]: *empty

File: tests/entities/__snapshots__/externalAgentActions.test.ts.snap [hunk 3]
```diff
@@ -4,44 +4,7 @@ exports[`tickerExternalAgentActions should filter by caller DID 1`] = `
...
-        "palletName": "sto",
-      },
-      Object {
-        "__typename": "TickerExternalAgentAction",
-        "callerId": "0x0854afeca045161ef42ba9be94c973f65efc0e8532933500865a1fd655148f6c",
-        "eventId": "FundraiserClosed",
-        "palletName": "sto",
-      },
-    ],
+    "nodes": Array [],
```
Comment (polymath-eric) [inline tests/entities/__snapshots__/externalAgentActions.test.ts.snap:7]: *empty

File: tests/entities/__snapshots__/portfolioMovements.test.ts.snap [hunk 4]
```diff
@@ -4,18 +4,7 @@ exports[`portfolioMovements should return all Portfolio movements associated wit
...
-        "address": "bc18d7b0241882bcfc94b5fe0d520988f2a9f4050a2a1df58c4de14ee23f3915",
-        "amount": "100",
-        "assetId": "4TICKER",
-        "fromId": "0xc5e2d554233da63d509ee482dbeed0ddec94dc1d0b45ebfdcdc48bd0928222b1/0",
-        "id": "139/1",
-        "nodeId": "WyJwb3J0Zm9saW9fbW92ZW1lbnRzIiwiMTM5LzEiXQ==",
-        "toId": "0xc5e2d554233da63d509ee482dbeed0ddec94dc1d0b45ebfdcdc48bd0928222b1/1",
-      },
-    ],
+    "nodes": Array [],
```
Comment (polymath-eric) [inline tests/entities/__snapshots__/portfolioMovements.test.ts.snap:7]: *empty

File: tests/entities/__snapshots__/portfolios.test.ts.snap [hunk 5]
```diff
@@ -34,18 +24,7 @@ exports[`portfolios should return all numbered Portfolio held by an Identity 1`]
...
-        "createdBlockId": "135",
-        "custodianId": null,
-        "id": "0xc5e2d554233da63d509ee482dbeed0ddec94dc1d0b45ebfdcdc48bd0928222b1/1",
-        "identityId": "0xc5e2d554233da63d509ee482dbeed0ddec94dc1d0b45ebfdcdc48bd0928222b1",
-        "name": "4_portfolio",
-        "nodeId": "WyJwb3J0Zm9saW9zIiwiMHhjNWUyZDU1NDIzM2RhNjNkNTA5ZWU0ODJkYmVlZDBkZGVjOTRkYzFkMGI0NWViZmRjZGM0OGJkMDkyODIyMmIxLzEiXQ==",
-        "number": 1,
-      },
-    ],
+    "nodes": Array [],
```
Comment (polymath-eric) [inline tests/entities/__snapshots__/portfolios.test.ts.snap:27]: *empty

File: tests/entities/__snapshots__/transferManagers.test.ts.snap [hunk 6]
```diff
@@ -4,16 +4,7 @@ exports[`transferManagers should return all transfer managers for a particular a
...
-      Object {
-        "__typename": "TransferManager",
-        "exemptedEntities": Array [],
-        "id": "15TICKER/Count/10",
-        "ticker": "15TICKER",
-        "type": "Count",
-        "value": 10,
-      },
-    ],
+    "nodes": Array [],
```
Comment (polymath-eric) [inline tests/entities/__snapshots__/transferManagers.test.ts.snap:7]: *empty

File: tests/setup.ts [hunk 7]
```diff
@@ -29,14 +29,17 @@ const retry = async <T>(
   }
   throw err;
 };
-const WAIT_UNTIL_BLOCK = 1000;
-export = async (): Promise<void> => {
+
+const WAIT_UNTIL_BLOCK = 450;
```
Comment (polymath-eric) [inline tests/setup.ts:33]: should query chain to see the latest

File: jest.config.js [hunk 8]
```diff
@@ -1,13 +1,14 @@
...
 module.exports = {
   globals: {
     'ts-jest': {
       tsconfig: 'tsconfig.test.json',
     },
   },
 
-  globalSetup: './tests/setup.ts',
-  globalTeardown: './tests/teardown.ts',
+  // globalSetup: './tests/setup.ts',
```
Comment (polymath-eric) [inline jest.config.js:10]: * clean up

**Comment by polymath-eric** [comment]
Blocked on getting remote snapshots in polymesh-local: https://github.com/PolymeshAssociation/polymesh-local/pull/46#pullrequestreview-1174819626

**Review by prashantasdeveloper — APPROVED** [review]

**Pull request merged**